numpydoc/hooks/validate_docstrings.py

```diff
@@ -357,10 +370,21 @@ def main(argv: Union[Sequence[str], None] = None) -> int:
         + "\n"
     )
 
-    parser = argparse.ArgumentParser(
-        description="Run numpydoc validation on files with option to ignore individual checks.",
-        formatter_class=argparse.RawTextHelpFormatter,
+    description = (
+        "Run numpydoc validation on files with option to ignore individual checks."
     )
+    if parent is None:
```

Comment:
I guess this is the part that confused me; why have two ways of adding the subparser? And, worse, without a parent automatically add lint subparser?